app/Config/Schema/schema.xml

```diff
@@ -44,7 +44,37 @@
       <unique />
     </index>
   </table>
-  
+
+  <table name="configuration_labels">
+    <field name="id" type="I">
+      <key />
+      <autoincrement />
+    </field>
+    <field name="co_id" type="I">
+      <constraint>REFERENCES cm_cos(id)</constraint>
+    </field>
+    <field name="label" type="C" size="64" />
+    <field name="color" type="C" size="8" />
```

Comment:
The color picker can include an alpha channel resulting in strings such as #a33937a4 - so size="8" is not enough.

Reply:
Thank you @spartanicus . I fixed the bug